controllers/operands/kubevirt.go

```diff
@@ -751,6 +748,10 @@ func hcoConfig2KvConfig(hcoConfig hcov1beta1.HyperConvergedConfig, infrastructur
 func getFeatureGateChecks(featureGates *hcov1beta1.HyperConvergedFeatureGates) []string {
 	fgs := make([]string, 0, 2)
 
+	if featureGates.DownwardMetrics == nil || (featureGates.DownwardMetrics != nil && *featureGates.DownwardMetrics) {
```

Comment:
I think it's too verbose. Can be done by:
```suggestion
	if featureGates.DownwardMetrics == nil || *featureGates.DownwardMetrics {
```

Reply:
Done.